Images/ai-images.py

```diff
@@ -15,10 +15,12 @@ def image_generate(prompt: str):
 
         url = "https://api.openai.com/v1/images/generations"
         TOKEN = get_token()
-        headers = {
-            "Content-Type": "application/json",
-            "Authorization": f"Bearer {TOKEN}",
-        }
+        headers = {"Content-Type": "application/json"}
+        # If the model endpoint is Azure, we need to use a different header
+        if "azure.com" in url:
+            headers["api-key"] = TOKEN
+        else:
+            headers["Authorization"] = f"Bearer {TOKEN}"
```

Comment:
```suggestion
       # otherwise default to the standard header format for openai
        else:
            headers["Authorization"] = f"Bearer {TOKEN}"
```